sphinx/writers/latex.py

```diff
@@ -1513,30 +1524,33 @@ def visit_entry(self, node):
                        '\\vbox{\\hbox{\\strut}}\\end{varwidth}%\n') + context
             self.needs_linetrimming = 1
         if len(node.traverse(nodes.paragraph)) >= 2:
-            self.table.has_oldproblematic = True
+            for i in range(cell.col, cell.col + cell.width):
+                self.table.empirical_colwidths[i] = max(self.table.empirical_colwidths[i], 17)
```

Comment:
admittedly I have not yet tested this works well with merged cells

Reply:
OMG, initially I used only integers in empirical_colwidths, so 17 made sense as next one after 16. But know they are floats. So I must fix this (will do later, if idea of this PR is accepted)